### PR TITLE
added a script to migrate local db

### PR DESCRIPTION
**Pull request by roy-sagar11**
* Added a script to migrate local db
* Added the necessary env in docker compose
* Added missing network in docker compose dynamo service
* Added a make command to run migration on running container

### CODE REVIEW

**Review by matbusby-fw — APPROVED** [review]
Nice enhancement!

File: api/scripts/createTable.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import AWS from 'aws-sdk';
```
Comment (fwpushan) [inline api/scripts/createTable.ts:1]: Overall looks good to me. But is there any way we can check if table exists or not. If not exists then create. But may be this is not required for MVP but I will suggest you to put a @todo for future dev

**Review by fwpushan — APPROVED** [review]
LGTM 👍

File: api/scripts/createTable.ts (hunk 1 above)
Comment (roy-sagar11) [reply api/scripts/createTable.ts:1]: awesome. I'll update it

**Pull request merged**